chainer/links/normalization/batch_normalization.py

```diff
@@ -33,15 +33,20 @@ class BatchNormalization(link.Link):
     fine-tuning mode.
 
     Args:
-        size (int or tuple of ints): Size (or shape) of channel
-            dimensions.
+        size (int, tuples of ints or None): Size (or shape) of channel
+            dimensions. If ``None``, the size will be determined from an axis
+            or axes of channel dimensions given by ``axis`` during the first
+            forward pass.
         decay (float): Decay rate of moving average. It is used on training.
         eps (float): Epsilon value for numerical stability.
         dtype (numpy.dtype): Type to use in computing.
         use_gamma (bool): If ``True``, use scaling parameter. Otherwise, use
             unit(1) which makes no effect.
         use_beta (bool): If ``True``, use shifting parameter. Otherwise, use
             unit(0) which makes no effect.
+        axis (int or tuple of ints): Axis or axes of channel dimension(s). When
+             ``size``` is ``None``, this value is used to determine parameter
+             sizes during the first forward pass.
```

Comment:
It does not say how this value is used when `size` is not `None`. So, how about adding "Otherwise, this value is not used"?